help.lisp

```diff
@@ -54,11 +54,44 @@
                                   (:right (format nil "~a~a~a" (if (= c 0) "" padstr) len s)))))))
     (apply 'mapcar 'concat (or cols '(nil)))))
 
+(defun sort-flattened-bindings (keymaps sort-pred)
+  "Returns all bindings in KEYMAPS in a flattened list, globally sorted with SORT-PRED."
+  (let ((all-bindings (flatten (mapcar #'kmap-bindings keymaps))))
```

Comment:
Would `alexandria:mappend` (akka flatmap) be an appropiate replacement to (flatten (mapcar ...))?